digital_comms/ccam.py

```diff
@@ -144,7 +246,16 @@ def demand(self):
         return summed_demand / summed_area
 
     def coverage(self):
-        """Return proportion of population with capacity coverage over a threshold
+        """Calculate coverage as the proportion of population with capacity coverage over a threshold
```

Comment:
"""Calculate coverage as the proportion of the population able to obtain the specified capacity threshold"""

Reply:
Done